paper/benchmarkset.tex

```diff
@@ -745,6 +745,102 @@ \subsubsection{Computational challenges posed by the T4 lysozyme benchmarks}
+Amberff14SB force-field. 
+
+The ligand binding free energies relative to the metastable apo closed state of BRD4(1) are calculated by including a simple restraint-wall scheme on the Asp88 $\psi$ dihedral, as described in 
+Ref.~\cite{Heinzelmann:2017:J.Chem.TheoryComput.}, so they are properly compared for each benchmark molecule. Alternatively, to determine the absolute ligand binding free energies relative to the apo open state, 
+the free energy of the apo closed-to-open transition can also be determined. As shown by Heinzelmann et al.~\cite{Heinzelmann:2017:J.Chem.TheoryComput.}, this transition works through a double hinge mechanism centered on the 
+Asp88 and Asp96 residues, and it is tractable using a single reaction coordinate (the Asp88 $\psi$ dihedral), making it suitable for the application, testing and comparison of various enhanced sampling methods. 
+
+\subsubsection{Possible side-chain rearrangements near the binding site}
+
+Even with the ZA-loop transition described above, 350ns simulations of the apo BRD4(1) did not exhibit the slow side-chain displacements of the conserved Tyr97 and Tyr139 residues (Fig. 3C), 
+which are part of the acetyl-lysine binding site and are located on the ZA and BC-loops, respectively. The side-chain movements of these two residues were first observed on simulations of the apo CREBBP 
+bromodomain~\cite{Steiner:2013:FEBSLett., Spilio:2014:Isr.J.Chem.}, with the Tyr1167 residue (equivalent to Tyr139 on BRD4(1)) adopting an outward conformation, followed by the occupation of the binding site by the side-chain of 
+Tyr1125 (Tyr97 on BRD4(1)). On the other hand, as in Refs.~\cite{Steiner:2013:FEBSLett., Spilio:2014:Isr.J.Chem.}, rotations on the Asn140 $\chi_2$ dihedral frequently take place in simulations of the apo BRD4(1), but they are found to
+be reversible and happening in a nanosecond time scale (Fig. 3D). Still, we cannot discard that slower side-chain transitions might still occur on BRD4(1) in the absence of the ligand, which would have to be accounted for if one is looking
+for gold standard results for this system.
+
+
+\subsubsection{Binding free energy calculations on BRD4(1)}
+
+Many studies have focused on the calculation of binding free energies of small molecules to BRD4(1), using a variety of methods such as alchemical transformations~\cite{aldeghi_accurate_2016, Aldeghi:2017:J.Am.Chem.Soc.}, 
+attach-pull-release (APR)~\cite{Heinzelmann:2017:J.Chem.TheoryComput.}, adaptive biasing potentials (ABP)\cite{Dickson:2016:J.Chem.Phys.}, steered molecular dynamics (SMD)~\cite{ Muvva:2014:Mol.Biosyst.}, as well as molecular 
+mechanics combined with either Poisson-Boltzmann (MM-PBSA)~\cite{Su:2017:J.Biomol.Struct.Dyn.,Deepak:2017:Chem.Biol.Drug.Des.} or generalized Born (MM-GBSA) surface area~\cite{Raj:2016:J.Biomol.Struct.Dyn.}. The first two methods have 
```

Comment:
Not really, just thought it would be interesting to mention it, but nothing really relevant there,